hs_core/hydroshare/users.py

```diff
@@ -272,7 +141,7 @@ def update_account(user, **kwargs):
     user.groups = groups
     return user.username
 
-
+# Pabitra: This one seems to be not used anywhere. Can I delete it?
```

Comment:
@pkdash I think this can be deleted. Not sure whether there is a requirement for REST API to support user listing which has not implemented yet, though. My guess is most likely not, but not sure.